pages/memgraph-lab/querying.mdx

```diff
@@ -0,0 +1,296 @@
+This view is beneficial for understanding the structural and relational aspects
+of your data. 
+
+Besides different results views, you can see the [Summary](#query-summary)
+of the executed query, copy or download the results in the JSON, CSV or TSV
+format and summary in the JSON format. You can also go fullscreen for better
+visilibility of the results.
+
+<Callout>
+Table rows can be expanded to show additional information about entities.
+</Callout>
+
+### Graph results
+
+Memgraph Lab utilizes the
+[Orb library](https://github.com/memgraph/orb) created by the Memgraph team for
+high-performance graph visualizations, ensuring a smooth and interactive
+experience.
+
+![](/pages/data-visualization/querying/graph-results.png) 
+
+<Callout>
+When rendering a graph that exceeds the set rendering limits, which might take
+considerable amount of time to preview, you will be asked if you want to proceed
+with the graph visualization or switch to the data view. The rendering limit can
+be set in the **Settings** section.
+</Callout>
+
+{<h4 className="custom-header">Exploring the graph</h4>}
+
+When results are shown as a graph, you can click on each node or relationship to
+see additional information in a property sidebar. 
+
+![](/pages/data-visualization/querying/property-sidebar.png) 
+
+You can also **Expand** a node to see its
+relationships, **Collapse** a node to hide its relationships, or **Hide** the
+node from the canvas. Besides clicking on a node and then on the button, you can
+also expand and collapse nodes with a double-click.
+
+![](/pages/data-visualization/lab-user-manual/graph-results.png)
+
+{<h4 className="custom-header">Graph customization</h4>}
+
+To customize the appearance of the graph you can update the GSS in the [Graph
+Style editor](#graph-style-editor) or change the **Display options** in the
+property sidebar which you can access by clicking on a graph object you'd like
+to customize. 
+
+Display options offers you the ability to change the color or text on display
+for the selected graph object or all the graph objects with the same label or of
+the same type.
+
+In the bottom left corner, you can **Enable physics**, that is, make nodes
+interact with each other, by pulling away or closing in to one another,
+depending on the strength of the relationships between them.
+
+In the top right corner of the graph you can open **Graph Preferences** and set
+the collision radius that dictates the margin radius for each node from its
+center, the repel force that dictates how strongly nodes repel each other, and
+the link distance that dictates the minimum required distance between two
+connected nodes.
+
+![](/pages/data-visualization/lab-user-manual/physics.png)
+
+Another interesting feature you can use on graph data results is the map
+background. This feature automatically turns on when the result nodes have
+numeral `lat` and `lng` properties.
```

Comment:
We could also add:

---

If latitude and longitude data are not stored in the default `lat` and `lng` properties, you can use a Graph Style Script (GSS) to map alternative fields for location tracking. For example:

```
// nodes have .latitude and .longitude fields
Define(LATITUDE_FIELD, "latitude")
Define(LONGITUDE_FIELD, "longitude")

// In the case of numeric latitude and longitude properties, set them up for a switch to a map view
@NodeStyle And(IsNumber(Property(node, LATITUDE_FIELD)), IsNumber(Property(node, LONGITUDE_FIELD))) {
  latitude: Property(node, LATITUDE_FIELD)
  longitude: Property(node, LONGITUDE_FIELD)
}
```